modules/vba_documents/app/controllers/vba_documents/v1/uploads_controller.rb

```diff
@@ -34,7 +34,8 @@ def show
           submission.status = request.headers['Status-Override']
           submission.save
         else
-          submission.refresh_status! unless submission.status == 'expired'
+          Rails.logger.info 'remove me! The linter wants a line here'
```

Comment:
Is this PR ready for review or are you still debugging? Just noticed this line

Reply:
This is ready for review. These changes will be rolled back once the transition to GovCIO is complete.